carapace-server/src/main/java/org/carapaceproxy/server/mapper/StandardEndpointMapper.java

```diff
@@ -99,6 +104,11 @@ public void configure(ConfigurationStore properties) throws ConfigurationNotVali
         LOG.info("configured mapper.forcedirector.parameter=" + forceDirectorParameter);
         this.forceBackendParameter = properties.getProperty("mapper.forcebackend.parameter", forceBackendParameter);
         LOG.info("configured mapper.forcebackend.parameter=" + forceBackendParameter);
+        // To add custom debugging header for request choosen mapping-path
+        this.debuggingHeaderEnabled = Boolean.parseBoolean(properties.getProperty("mapper.debug", "false"));
+        LOG.info("configured mapper.debug=" + debuggingHeaderEnabled);
+        this.debuggingHeaderName = "X-" + properties.getProperty("mapper.debug.name", DEBUGGING_HEADER_DEFAULT_NAME);
```

Comment:
this way you are forcing that prefix even on the user-configured value.
It is better to have a default of "X-Proxy-Path" and let the user override the value